src/uu/sed/src/processor.rs

```diff
@@ -161,6 +166,21 @@ fn write_chunk(
     Ok(())
 }
 
+/// Return the RE or the saved RE if the RE is None.
+/// Update the saved RE to RE.
+fn re_or_saved_re(regex: &Option<Regex>, context: &mut ProcessingContext) -> UResult<Regex> {
+    match regex {
+        Some(re) => {
+            *context.saved_regex.borrow_mut() = Some(re.clone());
```

Comment:
would it be possible to ignore the clone ?

Reply:
Indeed; fixed.

Reply:
impressive!